decidim-core/app/types/decidim/core/user_entity_input_filter.rb

```diff
@@ -46,6 +46,16 @@ class UserEntityInputFilter < BaseInputFilter
                    op_name.or(op_nick)
                  end
                end
+      argument :exclusion,
```

Comment:
```suggestion
      argument :exclude_ids,
```

I'd change this for its imperative form, and being more specific about what's been used to exclude